### ISSUE TITLE
python 3.x

### ISSUE THREAD

**Issue by koenedaele** [OnroerendErfgoed/augeias#5]
Looks like PairTree is only available for python 2.7. And not the best maintained library out there. This means that python3 compatibility will be hard to achieve.

Either we need to fork PairTree and fix it ourselves or not ship it as the default environment?

**Comment by koenedaele** [comment]
Found a GitHub repo for PairTree that has a Python 3 Pull Request: benosteen/pairtree#2

**Comment by koenedaele** [comment]
Another PairTree implementation (but very minimal): https://github.com/edsu/ptree

**Comment by koenedaele** [comment]
We'll need to fork PairTree ourselves and make it py3 compatible.

**Comment by cecemel** [comment]
Is it ok if I fork it and rename it OEPairtree, and start at 0.1.0? I guess this will be the least confusing path.

**Comment by cecemel** [comment]
update
--------
So, I've forked the pairtree repo into  [oepaitree](https://github.com/OnroerendErfgoed/oepairtree) (thx @cedrikv )

I merged the PR suggested here. (perhaps a little soon, but it didn't look that bad)

Started writing some additional tests (some basic sanity checks) and immediately two (critical) bugs showed up. 
See: https://github.com/OnroerendErfgoed/oepairtree/issues/3 and https://github.com/OnroerendErfgoed/oepairtree/issues/1

Also, I saw things in the library which really need a refactor.  (see https://github.com/OnroerendErfgoed/oepairtree/issues/3)

question
---------
My gut feeling tells me, the way home is longer than initially expected. To you want me to go through with this for sprint 66?
@koenedaele
